rebound/__init__.py

```diff
@@ -57,11 +57,13 @@ class NoParticles(Exception):
     """The simulation exited because no particles are left in the simulation."""
     pass
     
-
+class ParticleNotFound(Exception):
+    """Particle was not found in the simulation."""
```

Comment:
Not super sure about the syntax. But doesn't this require a "pass" as for the ones above.